packages/datetime/src/dateRangePicker.tsx

```diff
@@ -554,7 +554,8 @@ export class DateRangePicker
 
     private updateRightView(rightView: MonthAndYear) {
         let leftView = this.state.leftView.clone();
-        if (!rightView.isBefore(rightView)) {
+        // a bug here
+        if (!leftView.isBefore(rightView)) {
```

Comment:
@chux0519 there's a pattern for the checks here, can we change to `!rightView.isAfter(leftView)`?

even better would be a helper function of `viewsInOrder(leftView, rightView)`